### PR TITLE
Recipient Details panel

### PR DESCRIPTION
**Pull request by ebdabbs**
**High level description:**
Adds a "Recipient Details" panel to the Award Summary "Additional Information" section.

**Technical details:**
- Creates a `RecipientDetails` component, since the generic `Accordion` component only handles plain text table data.
- Adds a new API field for parent internal id to the `BaseAwardRecipient` model.

**JIRA Ticket:**
[DEV-2740](https://federal-spending-transparency.atlassian.net/browse/DEV-2740)

**Mockup:**
https://bahdigital.invisionapp.com/share/GVIAAHOBER9#/296001079_Award_Summary_2-0_-_IDV_-future-

The following are ALL required for the PR to be merged:
- [x] Code review
- [x] Design review
- [x] [API contract](https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/awards/AwardProfile.md#recipient-object) updated
- [x] [API #1886](https://github.com/fedspendingtransparency/usaspending-api/pull/1886) merged
- [x] [API #1891](https://github.com/fedspendingtransparency/usaspending-api/pull/1891) merged
- [x] Verified cross-browser compatibility

### CODE REVIEW

File: src/js/models/v2/awardsV2/BaseAwardRecipient.js [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
-
 import CoreLocation from 'models/v2/CoreLocation';
 
 const BaseAwardRecipient = {
     populate(data) {
         this.internalId = data.recipient_hash || '';
         this.name = data.recipient_name || 'Unknown';
         this.duns = data.recipient_unique_id || '--';
-        this.parentName = data.recipient_parent_name || '--';
+        this.parentName = data.parent_recipient_name || '';
```
Comment (jonhill13) [inline src/js/models/v2/awardsV2/BaseAwardRecipient.js]: Did we add this field to the mock API?

File: src/js/models/v2/awardsV2/BaseAwardRecipient.js [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
         this.parentDuns = data.parent_recipient_unique_id || '--';
+        this.parentInternalId = data.parent_recipient_hash || '';
```
Comment (jonhill13) [inline src/js/models/v2/awardsV2/BaseAwardRecipient.js]: Did we add this field to the mock API?

**Comment by jonhill13** [comment]
@ebdabbs 
Screen Shot 2019-06-27 at 3 39 09 PM

File: src/js/models/v2/awardsV2/BaseAwardRecipient.js (hunk 1 above)
Comment (ebdabbs) [reply src/js/models/v2/awardsV2/BaseAwardRecipient.js]: Added to the mock API response we use in unit tests. It's also been added to the [API Contract](https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/awards/AwardProfile.md#recipient-object).

**Pull request merged**